doc/sphinx/addendum/nsatz.rst

```diff
@@ -34,6 +34,12 @@ Nsatz: tactics for proving equalities in integral domains
 
    You can load the ``Nsatz`` module with the command ``Require Import Nsatz``.
 
+   Alternatively, if you prefer to not transitively depend on the
+   files declaring the axioms used to define the real numbers, you can
+   ``Require Import NsatzTactic`` instead; this will still allow
+   :tacn:`nsatz` to solve goals defined about :math:`\mathbb{Z}`,
+   :math:`\mathbb{Q}`, and any user-registered rings.
```

Comment:
```suggestion
   Alternatively, if you prefer not to transitively depend on the
   files declaring the axioms used to define the real numbers, you can
   ``Require Import NsatzTactic`` instead; this will still allow
   :tacn:`nsatz` to solve goals defined about :math:`\mathbb{Z}`,
   :math:`\mathbb{Q}` and any user-registered rings.
```

Is there a simpler phrase than "declaring the axioms used to define"?

Reply:
Not that I'm aware of